### PR TITLE
Clean up files from linters and validators

### PR DESCRIPTION
**Pull request by BradEstacio**
Just cleaning up a bunch of files to get rid of any potential warnings and errors from the linter files and validators.

### CODE REVIEW

**Comment by AlirpJ** [comment]
In site.js I think line 306 cartItems should be cart-items like the rest

**Comment by BradEstacio** [comment]
@AlirpJ It is you were probably looking at the file before the changes.

File: payment/index.html [hunk 1]
```diff
@@ -1,93 +1,93 @@
...
+        <li><a class="nav-button" id="cart-nav" href="../cart/">Cart</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main id="main">
+    <a href="../cart/" class="custom-button" id="custom-button">Back to Cart</a>
+
+    <aside id="summary">
+      <form name="cartItems">
```
Comment (AlirpJ) [inline payment/index.html]: This part should be cart-items?

**Review by AlirpJ — APPROVED** [review]
yeppers peppers!

**Review by pparagonn — APPROVED** [review]

**Pull request merged**